doc/plugins/tenant/common/development/codification.md

```diff
@@ -14,14 +14,25 @@ modellerine ilişkin kodifikasyon lojiği bulunmaktadır.
 `Unit`
 ------
 
-*`code_generator(starting:, ending:, pattern: nil, memory:)`*
+*`code_generator(starting:, ending:, only: nil, except: nil, memory:)`*
```

Comment:
PR'ın amacına uygun olmayan bir review yapayım. Burada `code_generator` kullanılmış, kullanıcı adı üretilirken `name_generate`, yani birinde `generator` diğerinde `generate`. Bunlar kasıtlı verilen isimler miydi?

Bunu sadece meraktan sordum, herhangi bir değişiklik yapılmasını istediğim için değil.

Reply:
Kasıt yok, review de haklısın.  Approve yapma sözü verirsen isimlendirmeyi bir sonraki tidyup'ta elden geçirme sözü veririm, şimdi üşendim. :-P

Reply:
Approve yapıldı hocam :grimacing:

Reply:
> Veritabanı güncelleme işleminin kim tarafından nasıl yapılacağı açıkca belirtilmiyor.

bunun ayrıca iş kaydını açarak assign edebiliriz. bununla ilgili bir Rake task var diye hatırlıyorum.